pkg/server/registry/service.go

```diff
@@ -53,6 +54,14 @@ func (s *LocalContainerRegistry) Start() error {
 		return err
 	}
 
+	if _, err := exec.LookPath("docker"); err != nil {
+		return fmt.Errorf("cannot find Docker. Please install and try again")
```

Comment:
Modify the message to be `cannot find Docker installation. Please install by following https://docs.docker.com/engine/install/ and try again.`